homework/src/index.js

```diff
@@ -1,47 +1,113 @@
+                elem.setAttribute(key, options[key]);
+            };
+        });           
+        return elem;
+    }
 
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+    const URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
+    
+    function renderRepository(container, repository){      
+        container.innerHTML='';
+        const table1 = createAppend('table',container,{id:"table1"});
+        const tBody1 = createAppend('tbody',table1);
+        const tr = createAppend('tr',tBody1);
+        const tr1 = createAppend('tr',tBody1);
+        const tr2 = createAppend('tr',tBody1);
+        const tr3 = createAppend('tr',tBody1);
+        const nametagTd = createAppend('td', tr, { text:"Repository: ", class:"tableData" });
+        const nameTag= createAppend('a', nametagTd, {text: repository.name});
+        nameTag.setAttribute('href', repository.html_url);
+        nameTag.setAttribute('target', '_blank');
+        createAppend('td', tr1, { text:"Description: ", class:"tableData" });
+        createAppend('td',tr1, {text: repository.description});
+        createAppend('td', tr2, { text:"Fork: ", class:"tableData" });
+        createAppend('td',tr2, {text: repository.forks_count});    
+        createAppend('td', tr3, { text:"Updated: ", class:"tableData" });
+        let updateDate = new Date (repository.updated_at);
+        createAppend('td',tr3, {text: updateDate.toLocaleString()});
+    }
 
-  window.onload = () => main(HYF_REPOS_URL);
+    async function createContainer (container, repository){
+        container.innerHTML='';
+        const contributorsURL = repository['contributors_url'];
+        createAppend('p', container, { text: 'Contributions', class:"tableData" });
+        try{
+            const contributors = await fetchJSON(contributorsURL)
+            const contTable = createAppend('table', container);
+            const contTbody = createAppend('tbody', contTable);
+            contributors.forEach(contributor => {
+                const contLink = contributor.html_url;
+                const contImg = contributor.avatar_url;
+                const contName = contributor.login;
+                const contNumber = contributor.contributions;
+                const contTr = createAppend('tr', contTbody);
+                const contributorsLink = createAppend('a', contTr, { href: contLink, target: '_blank', class: 'contributorsLink' });
+                const contTr1 = createAppend('tr', contributorsLink, { id: `${contributor['id']}`, class: 'contributorsTr' });
+                const tdImg = createAppend('td', contTr1, { id: `${contName}Td`});
+                createAppend('img', tdImg, { src: contImg, alt: `picture of ${contName}`, class: 'cont-images' });
+                createAppend('td', contTr1, { text: contName, class: 'cont-info cont-name' });
+                createAppend('td', contTr1, { text: contNumber, class: 'cont-info cont-num' });
```

Comment:
As per my previous feedback, the `cont-info` class is not used for anything. The logical conclusion I expected you to draw is to remove it altogether. Declaring things that are unused puts doubt in the mind of the reviewer.